cmd/plugin/rpaasv2/cmd/autoscale.go

```diff
@@ -0,0 +1,309 @@
+	client, err := getClient(c)
+	if err != nil {
+		return err
+	}
+
+	updateArgs := rpaasclient.UpdateAutoscaleArgs{
+		Instance:    c.String("instance"),
+		MinReplicas: int32(c.Int("minReplicas")),
+		MaxReplicas: int32(c.Int("maxReplicas")),
+		CPU:         int32(c.Int("cpu")),
+		Memory:      int32(c.Int("memory")),
+	}
+	_, err = client.UpdateAutoscale(c.Context, updateArgs)
+	if err != nil {
+		return err
+	}
+
+	fmt.Fprintf(c.App.Writer, "Autoscale of %s successfully updated\n", formatInstanceName(c))
+	return nil
+}
+
+func NewCmdGetAutoscale() *cli.Command {
+	return &cli.Command{
+		Name:  "get",
+		Usage: "Retrieves autoscale configuration of the desired instance",
+		Flags: []cli.Flag{
+			&cli.StringFlag{
+				Name:    "service",
+				Aliases: []string{"tsuru-service", "s"},
+				Usage:   "the Tsuru service name",
+			},
+			&cli.StringFlag{
+				Name:     "instance",
+				Aliases:  []string{"tsuru-service-instance", "i"},
+				Usage:    "the reverse proxy instance name",
+				Required: true,
+			},
+			&cli.BoolFlag{
+				Name:    "raw-output",
+				Aliases: []string{"r", "raw"},
+				Usage:   "show as JSON instead of go template format",
+				Value:   false,
+			},
+		},
+		Before: setupClient,
+		Action: runGetAutoscale,
+	}
+}
+
+func runGetAutoscale(c *cli.Context) error {
+	client, err := getClient(c)
+	if err != nil {
+		return err
+	}
+
+	args := rpaasclient.GetAutoscaleArgs{
+		Instance: c.String("instance"),
+		Raw:      c.Bool("raw-output"),
+	}
+
+	spec, _, err := client.GetAutoscale(c.Context, args)
+	if err != nil {
+		return err
+	}
+
+	if args.Raw {
+		return writeAutoscaleJSON(c.App.Writer, spec)
+	}
+
+	if spec != nil {
+		writeAutoscale(c.App.Writer, spec)
+	}
+
+	return nil
+}
+
+func NewCmdRemoveAutoscale() *cli.Command {
+	return &cli.Command{
+		Name:  "remove",
+		Usage: "Removes autoscale of the desired instance",
+		Flags: []cli.Flag{
+			&cli.StringFlag{
+				Name:    "service",
+				Aliases: []string{"tsuru-service", "s"},
+				Usage:   "the Tsuru service name",
+			},
+			&cli.StringFlag{
+				Name:     "instance",
+				Aliases:  []string{"tsuru-service-instance", "i"},
+				Usage:    "the reverse proxy instance name",
+				Required: true,
+			},
+		},
+		Before: setupClient,
+		Action: runRemoveAutoscale,
+	}
+}
+
+func runRemoveAutoscale(c *cli.Context) error {
+	client, err := getClient(c)
+	if err != nil {
+		return err
+	}
+
+	args := rpaasclient.RemoveAutoscaleArgs{
+		Instance: c.String("instance"),
+	}
+
+	_, err = client.RemoveAutoscale(c.Context, args)
+	if err != nil {
+		return err
+	}
+
+	fmt.Fprintf(c.App.Writer, "Autoscale of %s successfully removed\n", formatInstanceName(c))
+
+	return nil
+}
+
+func writeAutoscaleJSON(w io.Writer, spec *clientTypes.Autoscale) error {
+	message, err := json.MarshalIndent(spec, "", "\t")
+	if err != nil {
+		return err
+	}
+
+	fmt.Fprintln(w, string(message))
+	return nil
+}
+
+func writeAutoscale(w io.Writer, autoscale *clientTypes.Autoscale) {
```

Comment:
I think you can avoid duplication reusing `writeAutoscaleOnTableFormat` func instead creating a new one.